app/scripts/ViewportTrackerVertical.js

```diff
@@ -146,6 +145,21 @@ class ViewportTrackerVertical extends SVGTrack {
 
     this.draw();
   }
+
+  setDimensions(newDimensions) {
+    super.setDimensions(newDimensions);
+
+    const xRange = this._xScale.range();
+    const yRange = this._yScale.range();
+    const yDiff = yRange[1] - yRange[0];
+    this.brush.extent([
```

Comment:
What is this doing?

Reply:
In `ViewportTracker2D`, the brushing extent was being set to `Number.MAX_VALUE` 
https://github.com/higlass/higlass/pull/864/files#diff-d04084b0801ca6d78773229b4e54ecffL30

which was causing issues when the rectangle was rendered, since the HTML attribute values (width, height, etc.) were on the order of 10^300.

On the other hand, in `ViewportTrackerVertical` and `ViewportTrackerHorizontal`, the extent was not being set at all, and the default extent (https://github.com/d3/d3-brush/blob/master/src/brush.js#L116) was causing the selection rectangles to be either too short or too skinny (I think because the default extent is based on the container dimensions, and the container was smaller before the viewconfig was fully loaded).

This change (for example in `ViewportTrackerVertical`) uses the scale range to set the extent. It sets the extent to the total y range (total height of the view), plus extra (total height / 2) on the top and bottom, to allow the brushing selection to move out of the view on the top or bottom, since the current behavior in higlass allows the brush to be moved outside the view, and I wanted to keep that behavior.

Reply:
👍